help/destinations/catalog/advertising/Criteo.md

```diff
@@ -0,0 +1,143 @@
+| Export frequency | Streaming | Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-types.html?lang=en#streaming-destinations). |
+
+## Use cases
+
+To help you better understand how to use Criteo's destination, here are some goals that Adobe Experience Platform customers can achieve with Criteo.
+
+### Use case 1
+
+Get traffic
+
+Showcase your business with relevant product offers and flexible creatives. With intelligent product recommendations, your ads will automatically feature the products most likely to trigger visits and engagement. Flexible targeting allows you to build audiences from Criteo's commerce data set or from your own prospect lists and Adobe CDP segments
+
+### Use case 2
+
+Increase website conversions
+
+When visitors leave your website, remind them what they're missing with retargeting ads that increase conversions by showing special deals and hyper-relevant offers, wherever they go next. Connect your Adobe CDP segment to re-engage existing customers or target consumers similar to your most loyal shoppers.
+
+## Connect to Criteo
+
+Steps to connect are as follows:
+
+1. Login to Adobe experience platform and connect to Criteo's destination
+
+![Login](../../assets/catalog/advertising/criteo/criteo1.jpeg)
+
+
+2. You will be redirected to Criteo to authorize the connection. You may need to first login with your Criteo credentials :
+
+![Auth1](../../assets/catalog/advertising/criteo/criteo2.jpeg)
+
+![Auth2](../../assets/catalog/advertising/criteo/criteo3.jpeg)
+
+![Auth3](../../assets/catalog/advertising/criteo/criteo4.jpeg)
+
+
+
+3. Fill out the details below marked with \*:
+   1. Name : A name to help you recognize this destination in the future
+   2. Description : A description to help you identify this destination in the future
+   3. API Version: Preview
+   4. Advertiser ID : Your Criteo advertiser ID
+
+![Details](../../assets/catalog/advertising/criteo/criteo5.jpeg)
+
+## Activate segments to this destination
+
+Read [Activate profiles and segments to streaming segment export destinations](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/activate/activate-segment-streaming-destinations.html?lang=en) for instructions on activating audience segments to this destination.
+
+## Exported data
+
+The body of request sent by Adobe Destination Testing API is:
```

Comment:
The results of the Adobe testing API are not relevant to the end-users who will read this documentation. Instead, let's please add a few lines about how to validate in the Criteo platform that data has successfully come through (e.g.: segments are populating/visible in Criteo).

Reply:
Done.